packages/dd-trace/src/config.js

```diff
@@ -774,27 +373,166 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
 
     const defaults = this._defaults = {}
 
-    this._setValue(defaults, 'service', service)
+    this._setValue(defaults, 'appsec.blockedTemplateHtml', undefined)
+    this._setValue(defaults, 'appsec.blockedTemplateJson', undefined)
+    this._setValue(defaults, 'appsec.customRulesProvided', false)
+    this._setBoolean(defaults, 'appsec.enabled', undefined)
```

Comment:
why is one using `setValue()` and the other `setBoolea()` when they're both supposed to be bools ? can't all of the entries in `_applyDefaults()` just be all using `setValue()` anyway ? since we don't need any parsing ? It would also make it easier to sort them alphabetically

Reply:
that's an idea Diaz